docker/README.md

```diff
@@ -1,38 +1,54 @@
 This script is located at https://get.docker.com/ and has the following instructions at the top of the file:
 
     This script is meant for quick & easy install via:
-    $ curl -fsSL https://get.docker.com -o get-docker.sh
-    $ sh get-docker.sh
-
-This will install and prepare everything needed to build images and run them in standalone mode.
-Additionally, docker has a built-in container orchestration platform called "Docker Swarm". Swarm needs to be enabled to use any of the deployment example scripts in `deploy/`, this can be enabled by simply running:
-
-     $ docker swarm init 
-
-# Development
-In the `dev/` folder you will find an example staged Swarm stack that included everything needed to build and test sakai in development.
-
-GOTO [Development README](dev/)
-
-# Building
-In the `build/` folder you will find examples for building a docker image for Sakai. Before deployment, you need to build the docker image you will deploy, examples are provided for building from source or a binary release.
-
-GOTO [Building README](build/)
-
-# Deployment
-Deploying Sakai in Docker Swarm is easy, even with a full stack of supporting services.
-
-GOTO [Deployment README](deploy/)
+    $ `curl -fsSL https://get.docker.com -o get-docker.sh`
+    $ `sh get-docker.sh`
+
+# Building the Image
+There is currently one variant of this docker build, to build from source code.
+
+## From source (github tag/branch)
+The source build uses a multi-stage build, building an intermediate image with JDK and Maven in which to build Sakai, then building a Tomcat image using only the binary artifacts from the build container. This creates a smaller Sakai image that does not include Maven and all the build time libraries, source, etc.
+
+To build from source use these steps:
+ 1. From this folder
+ 1. Execute `docker build --build-arg release=master -t sakai -f ./Dockerfile.source .` substituting "master" for the branch/tag you wish to build
+ 1. Upon completion you can execute `docker image ls sakai` to verify it's creation
+    *     $ docker image ls sakai
+          REPOSITORY          TAG                 IMAGE ID            CREATED             SIZE
+          sakai        latest    ac16fc4db9dd   About a minute ago   1.86GB
+
+# Running this image
+
+## Starting MariaDB/MySQL
+You'll need to have MySQL/MariaDB started up first. The default config of the sakai.properties if for MariaDB on localhost.
+
+Here's a sample to startup MySQL from Docker
+```
+	# May want to include an opt for docker rm sakai-mariadb
+	# Start it if we've already created it, unless we want to re-create
+	docker run -p 127.0.0.1:53306:3306 -d --name="sakai-mariadb" --pull always \
+	    -e "MARIADB_ROOT_PASSWORD=sakairoot" \
+	    -v "./mysql/scripts:/docker-entrypoint-initdb.d" \
+	    -d mariadb:10 --lower-case-table-names=1 || docker start "sakai-mariadb"
+```
+## Starting up Sakai and linking to Maria
+    `docker run --rm -p 8080:8080 --name sakai-tomcat --link sakai-mariadb sakai`
```

Comment:
You might want to add instructions as to how to connect ot the database.  I did this:

    mysql -u root -P 53306 -p -h 127.0.0.1

Password `sakairoot`

Reply:
This was just in this SQL that I have in here as a default to run and prep the DB. Probably just some legacy I'd found in documentation somewhere.

https://github.com/sakaiproject/sakai/pull/13113/files#diff-ec7a841b95a0c9984fc69c019a3b384e7a7fade863b32ddc4e3c76b92506f278

I can clean up these unused databases from the script.

Reply:
The Maria bit was very nice to have.  Easy for you - harder for us docker beginners - thanks.

Reply:
I updated the documentation to add running mysql (Maria) directly from the container and also check that it's running and avoid creating the container if it already exists. 

I also updated the script to remove the old databases.